product_landing_page/index.css

```diff
@@ -62,3 +61,78 @@ body {
   .nav__item {
     margin: 2px auto;
   }
+}
+
+.container {
```

Comment:
Przydałby się tutaj padding horzytonalnie, bo istnieje rozdzielczość, dla której np. tekst z tagu `legend` nie ma "oddechu". I zamiast dodawać go w selektorze `.get-started-section__title`, lepiej go dodać w kontenerze, bo może się pojawić inny prócz `__title` przypadek, gdzie będzie trzeba to zrobić :) Proponuję coś w stylu `0.5rem`.
![header](https://user-images.githubusercontent.com/23278550/50372377-ddda0c00-05cd-11e9-8304-1b4e65d670cd.png)